### PR TITLE
Release v0.3.0 

### PR DESCRIPTION
**Pull request by MatthewMiddlehurst**
Release v0.3.0 preparation

### CODE REVIEW

File: docs/changelogs/v0.3.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+- [MNT] Remove the test_methods_have_no_side_effects for inceptionTime classifier from _config ({pr}`338`) {user}`hadifawaz1999`
+- [MNT] Fix workflow concurrency ({pr}`350`) {user}`MatthewMiddlehurst`
+- [MNT] Change `update_contributors.yml` to create PRs ({pr}`349`) {user}`MatthewMiddlehurst`
+- [MNT] Changes repo owner to aeon-toolkit in `.all-contributorsrc` ({pr}`359`) {user}`GuiArcencio`
+- [MNT] docs update to fix readthedocs fail ({pr}`386`) {user}`TonyBagnall`
+- [MNT] Add python 3.11 support ({pr}`191`) {user}`lmmentel`
+- [MNT] Remove any reference of pykalman ({pr}`380`) {user}`hadifawaz1999`
+- [MNT] Unstable extras ({pr}`365`) {user}`MatthewMiddlehurst`
+- [MNT] Remove the test_methods_have_no_side_effects for inceptionTime classifier from _config ({pr}`338`) {user}`hadifawaz1999`
+- [MNT] Cleanup the forecasting tests ({pr}`192`) {user}`lmmentel`
```
Comment (lmmentel) [inline docs/changelogs/v0.3.md:98]: This is duplicated from line 86, could be dropped :)

Comment (MatthewMiddlehurst) [reply docs/changelogs/v0.3.md:98]: this is actually from v0.2, but didnt notice it was duplicated! wonder why

Comment (MatthewMiddlehurst) [reply docs/changelogs/v0.3.md]: removed it from the old changelog and updated the new v0.3 one

**Review by TonyBagnall — APPROVED** [review]

**Review by lmmentel — APPROVED** [review]
LGTM!

**Pull request merged**